<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveStateImage"/>
    <addaction name="actionLoadStateImage"/>
    <addaction name="separator"/>
    <addaction name="actionFileScreenshot"/>
    <addaction name="actionFileScreenshotAs"/>
    <addaction name="actionFileScreenshotToClipboard"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>Emulator</string>
    </property>
    <addaction name="actionEmulatorRun"/>
    <addaction name="actionEmulatorReset"/>
    <addaction name="actionactionEmulatorAutostart"/>
    <addaction name="separator"/>
    <addaction name="actionSoundEnabled"/>
    <addaction name="separator"/>
    <addaction name="actionConfRam512"/>
    <addaction name="actionConfRam1024"/>
    <addaction name="actionConfRam2048"/>
    <addaction name="actionConfRam4096"/>
   </widget>
   <widget class="QMenu" name="menuDrives">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionDrivesFloppy0"/>
    <addaction name="actionDrivesFloppy1"/>
    <addaction name="separator"/>
    <addaction name="actionDrivesHard"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebugConsoleView"/>
    <addaction name="separator"/>
    <addaction name="actionDebugDisasmView"/>
    <addaction name="actionDebugMemoryView"/>
    <addaction name="separator"/>
    <addaction name="actionDebugStepInto"/>
    <addaction name="actionDebugStepOver"/>
    <addaction name="separator"/>
    <addaction name="actionDebugClearConsole"/>
    <addaction name="actionDebugRemoveAllBreakpoints"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewKeyboard"/>
    <addaction name="separator"/>
    <addaction name="actionViewMode0"/>
    <addaction name="actionViewMode1"/>
    <addaction name="actionViewMode2"/>
    <addaction name="actionViewMode3"/>
    <addaction name="actionViewMode4"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEmulator"/>
   <addaction name="menuDrives"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEmulatorRun"/>
   <addaction name="actionEmulatorReset"/>
   <addaction name="separator"/>
   <addaction name="actionDrivesFloppy0"/>
   <addaction name="actionDrivesFloppy1"/>
   <addaction name="separator"/>
   <addaction name="actionDrivesHard"/>
   <addaction name="separator"/>
   <addaction name="actionSoundEnabled"/>
   <addaction name="actionFileScreenshot"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEmulatorRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconRun.svg</normaloff>:/images/iconRun.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionEmulatorReset">
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconReset.svg</normaloff>:/images/iconReset.svg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDrivesFloppy0">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconFloppySlot.svg</normaloff>
     <normalon>:/images/iconFloppy.svg</normalon>:/images/iconFloppySlot.svg</iconset>
   </property>
   <property name="text">
    <string>Floppy 0</string>
   </property>
   <property name="iconText">
    <string>0</string>
   </property>
  </action>
  <action name="actionDrivesFloppy1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconFloppySlot.svg</normaloff>
     <normalon>:/images/iconFloppy.svg</normalon>:/images/iconFloppySlot.svg</iconset>
   </property>
   <property name="text">
    <string>Floppy 1</string>
   </property>
   <property name="iconText">
    <string>1</string>
   </property>
  </action>
  <action name="actionDrivesHard">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconHddSlot.svg</normaloff>:/images/iconHddSlot.svg</iconset>
   </property>
   <property name="text">
    <string>Hard Drive</string>
   </property>
   <property name="iconText">
    <string>HDD</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileScreenshot">
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconScreenshot.svg</normaloff>:/images/iconScreenshot.svg</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="iconText">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionDebugStepInto">
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconStepInto.svg</normaloff>:/images/iconStepInto.svg</iconset>
   </property>
   <property name="text">
    <string>Step Into</string>
   </property>
  </action>
  <action name="actionDebugStepOver">
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconStepOver.svg</normaloff>:/images/iconStepOver.svg</iconset>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
  </action>
  <action name="actionDebugDisasmView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disasm View</string>
   </property>
  </action>
  <action name="actionDebugMemoryView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Memory View</string>
   </property>
  </action>
  <action name="actionDebugConsoleView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconDebugger.svg</normaloff>:/images/iconDebugger.svg</iconset>
   </property>
   <property name="text">
    <string>Debug Views</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionViewMode0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>416 x 300 Debug</string>
   </property>
  </action>
  <action name="actionViewMode1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>624 x 450</string>
   </property>
  </action>
  <action name="actionViewMode3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1040 x 750</string>
   </property>
  </action>
  <action name="actionSoundEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtNeonBtl.qrc">
     <normaloff>:/images/iconSound.svg</normaloff>:/images/iconSound.svg</iconset>
   </property>
   <property name="text">
    <string>Sound Enabled</string>
   </property>
   <property name="iconText">
    <string>Sound</string>
   </property>
  </action>
  <action name="actionViewMode4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1248 x 900</string>
   </property>
  </action>
  <action name="actionSaveStateImage">
   <property name="text">
    <string>Save State...</string>
   </property>
  </action>
  <action name="actionLoadStateImage">
   <property name="text">
    <string>Load State...</string>
   </property>
  </action>
  <action name="actionViewKeyboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keyboard</string>
   </property>
  </action>
  <action name="actionFileScreenshotAs">
   <property name="text">
    <string>Save Screenshot as...</string>
   </property>
  </action>
  <action name="actionViewMode2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>832 x 600</string>
   </property>
  </action>
  <action name="actionFileScreenshotToClipboard">
   <property name="text">
    <string>Screenshot to Clipboard</string>
   </property>
  </action>
  <action name="actionactionEmulatorAutostart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autostart</string>
   </property>
  </action>
  <action name="actionDebugClearConsole">
   <property name="text">
    <string>Clear Console Log</string>
   </property>
  </action>
  <action name="actionDebugRemoveAllBreakpoints">
   <property name="text">
    <string>Remove All Breakpoints</string>
   </property>
  </action>
  <action name="actionConfRam512">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RAM 512 KB</string>
   </property>
  </action>
  <action name="actionConfRam1024">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RAM 1024 KB</string>
   </property>
  </action>
  <action name="actionConfRam2048">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RAM 2048 KB</string>
   </property>
  </action>
  <action name="actionConfRam4096">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RAM 4096 KB</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtNeonBtl.qrc"/>
 </resources>
 <connections/>
</ui>
